src/utilities/restService.ts

```diff
@@ -57,14 +76,18 @@ export async function restService<D>({
     const queryParams = createQueryParams(params);
     path = `${path}?${queryParams}`;
   }
-  return fetch(path)
+  return fetch(path, { headers })
     .then(async response => {
       const { status } = response;
 
       let data: undefined;
 
       try {
         data = await response.json();
+        const warning = response.headers.get('Warning');
+        if (warning) {
+          console.warn(warning);
```

Comment:
I'd call the `logError` function (`import { logError } from 'context/ErrorLogger';`), this way this gets reported in Sentry so we get notified that we need to update the endpoint used.